Source/Response.swift

```diff
@@ -77,4 +77,16 @@ public extension Response {
         }
         return string
     }
+
+    /// Maps received data at key path into a String.
+    public func mapString(atKeyPath keyPath: String?) throws -> String {
```

Comment:
Normally I would stand with @BasThomas, but in this case, where the implementation is different, I think we can use the solution proposed by @bchrobot.